New in doc: June (#3263)
ezsystems/developer-documentation: docs/resources/new_in_doc.md

This page contains recent highlights and notable changes in [[= product_name =]] documentation.

## June 2026

### Releases

- [v5.0.9 release notes](https://doc.ibexa.co/en/5.0/release_notes/ibexa_dxp_v5.0/#ibexa-dxp-v509)
- [v4.6.31 release notes](https://doc.ibexa.co/en/4.6/release_notes/ibexa_dxp_v4.6/#ibexa-dxp-v4631)

> - [3.3 as reached End of Life (EOL) on 30 June 2026](https://support.ibexa.co/Public/service-life). 3.3 documentation can still be accessed in the **Change version** drop-down menu by unfolding **Older versions**.

Reviewer: ```suggestion
- [Ibexa DXP 3.3 has reached End of Life (EOL) on 30 June 2026](https://support.ibexa.co/Public/service-life). You can access the 3.3 documentation by unfolding **Older versions** in the **Change version** drop-down menu.
```

> - [Ibexa DXP 3.3 has reached End of Life (EOL) on 30 June 2026](https://support.ibexa.co/Public/service-life). You can access the 3.3 documentation by unfolding **Older versions** in the **Change version** drop-down menu.

Author: ```suggestion
- [[[= product_name =]] 3.3 has reached End of Life (EOL) on 30 June 2026](https://support.ibexa.co/Public/service-life). You can access the 3.3 documentation by unfolding **Older versions** in the **Change version** drop-down menu.
```

### Users

- Updated [user authentication customization example](user_authentication.md)

### Background tasks and scheduled tasks

- Reworked documentation about [scheduled tasks](install_ibexa_dxp.md#schedule-tasks)

> - Described [Messenger configuration for production](background_tasks.md#configure-for-production-environment)

Reviewer: ```suggestion
- Described [Ibexa Messenger configuration for production](background_tasks.md#configure-for-production-environment)
```
